filters/filter_21_Annoyances_Other/metadata.json

```
@@ -0,0 +1,17 @@
+{
+  "filterId": 21,
+  "name": "AdGuard Other filter",
```

I am not entirely sure about the proposed name though.

```suggestion
  "name": "AdGuard Other Annoyances filter",
```

---

It's a draft name, that can be changed.